### DISCUSSION TITLE
Which  I/O Design Minio use

### DISCUSSION THREAD

**Discussion by gennady-voronkov** [minio/minio#13989] [Q&A] [answered]
Hi All,
Could You please clarify some questions:
1. Does Minio use direct_io=true during read/write? is it possible to turn off direct_io(direct_oi=false)?
2. What is chunk size? and chunk size is related with EC schema?

**Comment by harshavardhana** [comment, marked as answer]
O_DIRECT=true
ECBlockSize=1MiB

**Reply by gennady-voronkov** [reply]
Does it mean, if content(object) is less or equal than 1M it does not use EC schema?
I see some low performance on content with size 10K/100K/1M. is it related that we don't use buffer cache? or because we write small object  to the same device, where keeping metadata?

**Reply by harshavardhana** [reply]
> Does it mean, if content(object) is less or equal than 1M it does not use EC schema?

No everything is EC'ed
